client/src/components/button.scss

```diff
@@ -31,14 +38,11 @@
 }
 
 .button:disabled, .button_negative:disabled {
-	border: none;
-	background: $color-disabled;
-	color: $color-font-alt;
-	pointer-events: none;
+	@include disabled-button;
 }
 
 .button_clear:disabled {
-	@extend .button:disabled;
+	@include disabled-button;
```

Comment:
Had to do this because of a little stricter compilation in dart sass. Specifically, compound selectors can't be extended – http://bit.ly/ExtendCompound

Reply:
I compared the two css outputs using node-sass and dart sass for this section and it produces the same effect.